pkg/scheduler/scheduler.go

```diff
@@ -605,11 +605,25 @@ func (sched *Scheduler) scheduleOne() {
 			fwk.RunUnreservePlugins(pluginContext, assumedPod, scheduleResult.SuggestedHost)
 			sched.recordSchedulingFailure(assumedPod, err, SchedulerError, fmt.Sprintf("Binding rejected: %v", err))
 		} else {
-			klog.V(2).Infof("pod %v/%v is bound successfully on node %v, %d nodes evaluated, %d nodes were found feasible", assumedPod.Namespace, assumedPod.Name, scheduleResult.SuggestedHost, scheduleResult.EvaluatedNodes, scheduleResult.FeasibleNodes)
+			node, _ := sched.Cache().GetNodeInfo(scheduleResult.SuggestedHost)
+			klog.V(2).Infof("pod %v/%v is bound successfully on node %q, %d nodes evaluated, %d nodes were found feasible. Bound node resource: %q.", assumedPod.Namespace, assumedPod.Name, scheduleResult.SuggestedHost, scheduleResult.EvaluatedNodes, scheduleResult.FeasibleNodes, nodeResourceString(node))
+
 			metrics.PodScheduleSuccesses.Inc()
 
 			// Run "postbind" plugins.
 			fwk.RunPostbindPlugins(pluginContext, assumedPod, scheduleResult.SuggestedHost)
 		}
 	}()
 }
+
+// nodeResourceString returns a string representation of node resources.
+func nodeResourceString(n *v1.Node) string {
+	if n == nil {
```

Comment:
`|| n.Status == nil`?

Reply:
Status is a struct object who will have default values, see [code](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L4557)

Reply:
oh right, my bad.

Reply:
No worries :)